src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -192,10 +262,33 @@ which we divide into several categories:
 | ScanDate        | RECOMMENDED           | [string][]               | Date of scan in the default unit `"YYYY-MM-DD[Z]"`, with the Z indicator being optional for indicating UTC timezone (see [Units](../02-common-principles.md#units)).                                 |
```

Comment:
What about this one? I think it was never completely resolved, but at least we seemed all in favour of removing it. Maybe not relevant for this PR.

Reply:
Good point, thanks for the reminder --> That one can be dealt with in https://github.com/bids-standard/bids-specification/pull/798 Chris had some remarks about your suggestion that need to be addressed before we can proceed :-)

Reply:
Chris proposal looks good to me!